join-chat-to-teams-meeting/Chat Experience with File Sharing/client.js

```diff
@@ -117,7 +117,7 @@ async function renderReceivedMessage(event) {
 
     // Inject image tag for all image attachments
     var imageAttachmentHtml = event.attachments
-        .filter(attachment => attachment.attachmentType === "teamsImage")
+        .filter(attachment => attachment.attachmentType === "image" && !messages.contains(attachment.id))
         .map(attachment => renderImageAttachments(attachment))
         .join('');
     messagesContainer.innerHTML += imageAttachmentHtml;
```

Comment:
Is there a way we can avoid setting the innerHtml and instead injecting DOM nodes to represent the images?

Reply:
we should be able to, but we might need some code changes to all 3 quickstarts. I inherited this from original quickstart from Chat team. Feel free to proposal changes if you think it's blocking